andela/kissa-rc
# #137005189 Cancel Order Functionality

#### What does this PR do?
Enables an admin / user to cancel order placed by customers
#### Description of Task to be completed?
Admin / User should be able to cancel order
#### How should this be manually tested?
As an admin, when you click on "Orders" from the dashboard, you will see the list of orders. Click on an order and from the sliding menu of order details on the right hand side, scroll down and add a comment why you want to cancel the order, click on "Cancel Order" and confirm.
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
137005189

#### Screenshots (if appropriate)
![screen shot 2017-01-11 at 11 30 08 am](https://cloud.githubusercontent.com/assets/23452546/21845181/e7edf7c6-d7f1-11e6-9e2f-4183e15c0ee0.png)


![screen shot 2017-01-16 at 5 08 44 pm](https://cloud.githubusercontent.com/assets/23452546/21990736/36ee22f2-dc10-11e6-98d3-becd0dff3770.png)


#### Questions:
N/A
